plugins/workbench-resources/src/components/SpaceView.svelte

```diff
@@ -14,13 +14,12 @@
 -->
 
 <script lang="ts">
-  import core, { Class, Doc, Ref, Space, WithLookup } from '@anticrm/core'
+  import core,{ Class,Doc,Ref,Space,WithLookup } from '@anticrm/core'
```

Comment:
Maybe run `format` to autoupdate this line?

Reply:
fixed